source/2020-01-17-the-ember-times-issue-131.md

```diff
@@ -9,6 +9,8 @@ responsive: true
 <!--alex ignore waiters-waitresses-->
 Привет, Эмберисты! <span style="font-style: italic;">Hello, Emberistas!</span> 🐹
 
+Help improve Ember's autotracking and reactivity system 💬,
+
```

Comment:
<!--alex ignore waiters-waitresses-->

Reply:
```suggestion
<!--alex ignore waiters-waitresses-->
```

Reply:
I think the line break disables the alex linter exception from above again - we can either add it here instead or remove the line break I think